canonical-websites/www.ubuntu.com
# Convert privacy notices to markdown and new layout

## Done

- Updated all the privacy notices to markdown and the newer layout
- Will update copy docs post this work

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/data-privacy](http://0.0.0.0:8001/legal/data-privacy)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- The content of the pages will have changed little, some titles cleaned-up and meta-descirptions added, the layout should look more like http://0.0.0.0:8001/legal/data-privacy/user-survey

## Issue / Card

Fixes #4687
